src/battle_script_commands.c

```diff
@@ -8071,6 +8071,7 @@ static void Cmd_removeitem(void)
         gBattleStruct->usedHeldItems[gBattlerPartyIndexes[battler]][GetBattlerSide(battler)] = itemId; // Remember if switched out
 
     gBattleMons[battler].item = ITEM_NONE;
+    gBattleStruct->canPickupItem |= gBitTable[battler];
```

Comment:
```suggestion
    gBattleStruct->canPickupItem |= (1u << battler);
```
we are trying to move away from direct bitfield calls so could you change it like that across the pr?